packages/curve-ui-kit/src/themes/components.ts

```diff
@@ -43,7 +43,7 @@ export const createComponents = (design: DesignSystem): ThemeOptions['components
   },
   MuiDialog: {
     styleOverrides: {
-      paper: { [basicMuiTheme.breakpoints.down('tablet')]: { margin: SizesAndSpaces.Spacing.md.mobile } },
+      paper: { maxHeight: '100vh', [basicMuiTheme.breakpoints.down('tablet')]: { margin: 0 } },
```

Comment:
Isn't 0 the default? :thinking:
```suggestion
      paper: { maxHeight: '100vh' },
```

Reply:
No, there is another default, it was quite a lot of margin at that hence why I think 0.5rem was set initially

Reply:
aha that's why I've overridden it